app/src/main/java/org/mozilla/reference/browser/browser/BrowserFragment.kt

```diff
@@ -75,13 +92,28 @@ class BrowserFragment : Fragment(), BackHandler {
         return false
     }
 
+    override fun onRequestPermissionsResult(requestCode: Int, permissions: Array<String>, grantResults: IntArray) {
+        when (requestCode) {
+            PERMISSION_WRITE_STORAGE_REQUEST -> {
+                if ((grantResults.isNotEmpty() && grantResults[0] == PERMISSION_GRANTED) &&
+                        isStoragePermissionAvailable()) {
+                    // permission was granted, yay!
```

Comment:
Not sure whether that comment is actually helpful 🙃

Reply:
Easter egg 🥚  😆